src/components/Navbar/navbar.js

```diff
@@ -6,6 +6,15 @@ import Email from '../Icons/email'
 import './style.css'
 
 function Navbar() {
+  var links = document.querySelectorAll('.navbar-nav li')
```

Comment:
1. Create a method with a function just to collapse the dropdown.
2. Check for screen size and make sure navbar is collapsed. 
3. Add the function to onClick of all the lists in dropdown.